src/elements/pill.svelte

```diff
@@ -1,45 +1,61 @@
-<svelte:options immutable tag='v-pill' />
-
-<script lang='ts'>
-
-import cx from 'classnames';
-import { dispatcher } from '../lib/dispatch';
-import { addStyles } from '../lib/index';
-import { htmlToBoolean } from '../lib/boolean';
-
-export let value = ''; 
-export let removable = 'true';
-export let readonly: string;
-export let disabled: string;
-let isReadonly: boolean;
-let isDisabled: boolean;
-let isRemovable: boolean;
-$: isRemovable = htmlToBoolean(removable, 'removable');
-$: isReadonly = htmlToBoolean(readonly, 'readonly');
-$: isDisabled = htmlToBoolean(disabled, 'disabled');
-
-const dispatch = dispatcher();
-addStyles();
-
-const handleRemove = () => {
-  dispatch('remove', { value });
-};
-
+<svelte:options immutable tag="v-pill" />
+
+<script lang="ts">
+  import cx from 'classnames';
```

Comment:
Could we restore the original indentation? It'll make it a bit easier to parse the actual changes :)